openhtf/io/output/mfg_inspector.py

```diff
@@ -381,11 +382,14 @@ def locked_get(self):
     def locked_put(self, credentials):
       self._credentials = credentials
   # pylint: enable=invalid-name,missing-docstring
-
-  def __init__(self, user, keydata, token_uri=TOKEN_URI):
+  
+  def __init__(self, user, keydata, token_uri=TOKEN_URI,
+               error_message=None, filename_pattern_fail=None):
```

Comment:
`filename_pattern_fail` was pretty mysterious until I read the whole thing. Maybe there's a clearer way to say this that hints better at its use, like `fallback_filename_pattern`.